### ISSUE TITLE
Redirect Request: /scorecard to /performance-dashboard

### ISSUE THREAD

**Issue by kennedycollins** [department-of-veterans-affairs/va.gov-team#6928]
### Instructions
- Requests for redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track different steps.
- It is your responsibility to notify VA stakeholders of pending redirect.  

### Description
*This is a detailed description of the request. It should include background information and reason for the request.*
The ThoughtWorks team was asked to rebuild and refresh the VA.gov scorecard, which has been orphaned for ~18 months. As part of that work, we came to the understanding that "scorecard" wasn't the best name for what we were building, and so we will instead be calling it the VA.gov "Performance Dashboard". Ultimately, we intend to move this to a subdomain, but as an intermediate step to make it clear that this is a new thing, distinct from the old "scorecard", we'll be creating /performance-dashboard and displaying the new site there. 

### Requestor team info
[ThoughtWorks]@department-of-veterans-affairs/thoughtworks)
@kennedycollins product lead
@karlbrown-va tech principal
@rumyeahs product manager
@ffafara-tw tech lead

### Implementation date
Date new URL(s) will be live: 
Current plan is to push the new site live onto both /performance-dashboard and /scorecard on or around 3/25.

Please indicate when the redirect(s) needs to be implemented"
- [ ] On the same date the new URLs launch
- [x] Can happen within 1-5 business days after new URLs launch
- [ ] Other - Please indicate timing : 

### Redirects needed

Current URL  |  Redirect Destination (new URL) | Notes
--- | --- | ---
www.va.gov/scorecard  |  www.va.gov/performance-dashboard  | All child pages of `/scorecard` should be redirected to this new URL

### Definition of done
- [x] Above information is provided and issue is tagged and assigned appropriately - *@ requesting team*
- [x] All appropriate VA stakeholders are notified of pending redirect - *@ requesting team*
- [x] Redirect request is vetted and documented and implementation plan is clear - *@ Content & IA team*
- [x] Request is assigned to appropriate team for implementation - *@ Content & IA team*
- [ ] Implementation team completes work - *@ Implementation team*
- [ ] Implementation pushed live and redirect is validated in production - *@ all*
- [ ] Ticket is closed - *@ requesting team*

**Comment by karlbrown-va** [comment]
Is this the ticket just to create the redirects? And do we need to have another ticket to create the performance-dashboard URL?

**Comment by mnorthuis** [comment]
@lunascoop This redirect has been vetted and can be scheduled and assigned by your team for implementation.  Please note the launch date in the description. 

@kennedycollins Please be sure to notify @lunascoop via this ticket if any changes are made to the scheduled date so that team can implement the redirect accordingly.

**Comment by lunascoop** [comment]
@ncksllvn @kelsonic This is a redirect request outside of the Public Websites team. I am tentatively scheduling this redirect for **03/30**, after the page is live.

**Comment by ncksllvn** [comment]
PR open

**Comment by MickinSahni** [comment]
2020-03-25 PW Planning
- Waiting for page to go-live before we implement the redirect cc @kennedycollins

**Comment by ncksllvn** [comment]
Off-topic, but @karlbrown-va does this mean we can remove all of these Scorecard/Playbook assets and references?

https://github.com/department-of-veterans-affairs/vagov-content/tree/master/pages/playbook
https://github.com/department-of-veterans-affairs/vets-website/search?q=playbook&unscoped_q=playbook

**Comment by karlbrown-va** [comment]
Is that the stuff that ends up here:
https://www.va.gov/playbook/

If so, I'd say we haven't discussed that directly per my recollection, but it seems it might be worth considering, not just b/c of the scorecard refresh but more generally from a content refresh perspective.

@kennedycollins or @rumyeahs do you recall any discussions about the playbook and its relation to scorecard?

**Comment by ncksllvn** [comment]
Oh yeah, whoops, Scorecard is not the same thing as Playbook 🤦‍♂ Sorry, dumb question!

**Comment by karlbrown-va** [comment]
I mean,  it's a good point. It talks about "The new vets.gov intends to deliver both."

That whole page is somewhat dated, and could be refreshed to reflect reality of today - e.g. there is no vets.gov, a bunch of cool things have been done on VA.gov, and more is to come.

**Comment by MickinSahni** [comment]
2020-03-26 PW Standup
- 2 open PRs: one is the redirect, other is content change in footer (#6933)
- awaiting confirmation that the new scorecard page is live cc @kennedycollins @karlbrown-va

**Comment by MickinSahni** [comment]
> Hi, yeah you may want to leave this in a branch until the new URL is live. We are awaiting another PR for this which is here: [department-of-veterans-affairs/devops#6558](https://github.com/department-of-veterans-affairs/devops/pull/6558)
> 
> We also need to actually push the dashboard/scorecard to production, which should happen within a day or two. Will ping you here when we're ready for the new footers.



Hey team, checking in to see if the page is now live per the [timeline on 6933](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6933#issuecomment-604088603). cc @karlbrown-va @kennedycollins

**Comment by lunascoop** [comment]
@MickinSahni @kelsonic I am tentatively pushing the redirect date (**04/02**)to later this week since the page is not live. We can reschedule if the page goes live on or after that date.

**Comment by kelsonic** [comment]
Added `Disallow: /performance-dashboard` to robots.txt in [the current PR](https://github.com/department-of-veterans-affairs/vets-website/pull/12012/files#diff-f8924c60278e9103e7a46a872bd0e991R97)! I won't merge it until you give me the go-ahead @lunascoop! 🙂

**Comment by ncksllvn** [comment]
@karlbrown-va just to confirm, should `/performance-dashboard` be excluded from search engines? I believe `/scorecard` was added there only because it hasn't been touched since modernized VA.gov launched. It could be worth having it there if the product is not ready to be fully user-facing or if the product is inherently not something we want showing up in search results.

**Comment by kelsonic** [comment]
![image](https://user-images.githubusercontent.com/12773166/77926289-31241b00-7263-11ea-9d42-3e27f0cfb921.png)

This is the conversation that led to adding `Disallow: /performance-dashboard`. I suppose if we are excluding it from search results to reduce traffic to it, it probably doesn't make sense to include it in the footer? Is it a secret route (e.g. like an admin area)?

**Comment by karlbrown-va** [comment]
I'd like @KevinMHoffmanUSDS and @clantosswett and @chrisj-usds to respond on that point (removing it from the footer entirely). AFAIK it is *not* considered 'secret'.

**Comment by karlbrown-va** [comment]
> I believe `/scorecard` was added there only because it hasn't been touched since modernized VA.gov launched. 

@ncksllvn I don't think that's the case. This is where it was removed: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3419

**Comment by ncksllvn** [comment]
I stand corrected!

**Comment by mnorthuis** [comment]
I am not an SEO expert by any means, but I believe if we do not want the new performance dashboard to appear in search results, there are a couple of options depending on how we want people to be able to access it:
1) If we want to provide a link in the footer so users can get to it from the site, but want to hide it from search results, then we should use the disallow in combination with an on page noindex.  Without the noindex, bots will still index it, but the disallow will keep them from crawling it and getting a description, which will result in a very vague search result. It's a bit of a mixed signal...
2) If we do not want users to find it unless they are directed to it by being given the URL, then the link should be removed from the footer.

**Comment by MickinSahni** [comment]
2020-03-30 PW Standup
- This too; pushing change date until confirmation received that page is live.

**Comment by chrisj-usds** [comment]
Just skimming notes above: 
- Scorecard / performance dashboard are not secret. It can remain in the footer. 
- Both can be excluded from search in robots.txt since they are not valuable veteran- or external-user-facing tools. (Looks like Mikki's suggestion of disallow + no-index is best path here)

**Comment by karlbrown-va** [comment]
Roger that @chrisj-usds. we can add noindex directive to the top of the page.

**Comment by lunascoop** [comment]
@kelsonic Page is live. The redirect can be implemented if ready.

**Comment by kelsonic** [comment]
@lunascoop I just have the PR up for the footer link update! I think @ncksllvn has the PR for the redirects! https://github.com/department-of-veterans-affairs/devops/pull/6556

**Comment by kelsonic** [comment]
@lunascoop Merged the PR for the footer link, will be deployed later today! https://github.com/department-of-veterans-affairs/vets-website/pull/12012

**Comment by MickinSahni** [comment]
2020-04-01 PW Standup
- @karlbrown-va @kennedycollins we noticed that the new version of the performance dashboard is on staging -- https://staging.va.gov/performance-dashboard/ but not in prod yet so we're not implementing the redirect/footer changes. 
  - What's your guidance on whether the redirect/footer changes need to wait for the new performance dashboard? We're in a holding pattern until then.

**Comment by kennedycollins** [comment]
@MickinSahni We're hoping to get the new page live tomorrow. However, given that the new URL is already live (at https://www.va.gov/performance-dashboard/) and is showing the same thing as the current scorecard page, I don't see any reason why we can't go ahead and "move" the page and then have the content change later. Happy to discuss this further on slack if you would like!

**Comment by karlbrown-va** [comment]
Hi - my suggestion is that, if we do change this redirect, that we also play this card at the same time:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6933

That way, clicking on the footer link 'performance-dashboard' will bring you to 'performance-dashboard'.

then, when the content updates, it will also say 'performance dashboard'. We have a few minor edits to make before going live, but that should be done by this week.

**Comment by ncksllvn** [comment]
Thanks @karlbrown-va!

@MickinSahni I added the `blocked` label to indicate we're going to hold off on this indefinitely, until we're given the go-ahead from @karlbrown-va.

**Comment by kelsonic** [comment]
Any update on this? @karlbrown-va

**Comment by karlbrown-va** [comment]
sorry, still on hold. Am confident this will be completed this week and we can unblock this.

**Comment by karlbrown-va** [comment]
@kelsonic we are now live at va.gov/performance-dashboard and can play this issue now.

**Comment by lunascoop** [comment]
Validated and closing ticket.
